internal/cli/run/run.go

```diff
@@ -95,6 +96,17 @@ func init() {
 			log.WithError(err).Error("Failed to create the network row")
 			return err
 		}
+		if ctx.Config.Advanced.BouncerURL != "" {
+			ctx.Session.SetAvailableHTTPSBouncer(ctx.Config.Advanced.BouncerURL)
```

Comment:
We may want in the future to support non-https bouncers (ex. onion service bouncers) and we may have to adjust this code here. Perhaps the function call on `Session` can be `SetAvailableBouncer` or will we do the switch in here based on the target bouncer format?

Reply:
My suggestion: let's handle this problem in the future? It's not super clear to me what benefits an onion service would add to our users compared to the current situation where the engine already knows how to proxy any non measurement traffic (including `mlab-ns`) through any proxy that speaks socks5 (hence also including tor) and uses such proxy service to somehow reach our services exiting on the plain old internet and using https.

Reply:
Sounds reasonable. Just though I would bring it up.

Reply:
Yeah, this seems like something we want to have a ticket for. I'll create one.

Reply:
Here: https://github.com/ooni/probe-cli/issues/47